### ISSUE TITLE
Missing screenshots in Flex administration docs

### ISSUE THREAD

**Issue by Karmalakas** [getgrav/grav-learn#904]
Here are couple of pages with missing screenshots (404 errors)

- grav-learn/pages/08.advanced/01.flex/01.administration/docs.17.md
- grav-learn/pages/08.advanced/01.flex/01.administration/03.configuration/docs.17.md

**Comment by rhukster** [comment]
Yes we never got around to them. Its on our list.  These broken images serve as placeholders for us at the mo so we know what needs to be done.

**Comment by pmoreno-rodriguez** [comment]
Fixed in [PR1120](https://github.com/getgrav/grav-learn/pull/1120)

**Comment by Karmalakas** [comment]
That PR broke DOM
https://github.com/getgrav/grav-learn/pull/1120#pullrequestreview-2758018880

**Comment by rhukster** [comment]
fixed
